examples/platforms/simulation/platform-simulation.h

```diff
@@ -66,6 +66,8 @@ enum
     OT_SIM_EVENT_RADIO_RECEIVED     = 1,
     OT_SIM_EVENT_UART_WRITE         = 2,
     OT_SIM_EVENT_RADIO_SPINEL_WRITE = 3,
+    OT_SIM_EVENT_POST_CMD           = 4,
```

Comment:
not used?

Reply:
It is not introduced by `OTNS`. However it is defined in existing virtual time simulator in python code. I add `OT_SIM_EVENT_POST_CMD           = 4,` here so that there won't be a gap between 3 and 5.

Reply:
How about using the consistent enumeration name `OT_SIM_EVENT_POSTCMD`?

Reply:
I removed it because it's not introduced by `OTNS`